R/plot_carp.R

```diff
@@ -807,30 +825,35 @@ carp_path_plotly <- function(x,
           x = ~V1,
           y = ~V2,
           color = I(~final_cluster),
-          size = 3)
+          size = 3) %>%
+        add_text(data = plot_frame_init_adj,
+                 x = ~x_adj,
+                 y = ~y_adj,
+                 text = ~label,
+                 #size = label_size,
+                 inherit = FALSE) %>%
+        plotly::layout(xaxis = list(title = "V1"), yaxis = list(title = "V2")) %>%
```

Comment:
Can we make this show the feature being plotted instead of the fixed strings V1 / V2?

Reply:
I didn't mean to remove this line - I was thinking something more like `layout(xaxis = list(title = axis[1]), yaxis = list(title=axis[2]))`